src/main/java/lotto/domain/Lotto.java

```diff
@@ -0,0 +1,38 @@
+package lotto.domain;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Lotto {
+
+  public static final Money PRICE = new Money(1000);
+  public static final int NUMBER_SIZE = 6;
+  private final List<LottoNumber> lottoNumbers;
+
+  public Lotto(int[] numbers) {
+    this(Arrays.stream(numbers).mapToObj(LottoNumber::new).collect(Collectors.toList()));
```

Comment:
```suggestion
    this(
        Arrays.stream(numbers)
            .mapToObj(LottoNumber::new)
            .collect(Collectors.toList())
   );
```
이렇게 표현해주면 읽기가 더 수월할 것 같아요!